test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eux
+
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+test-build() {
+  nix-build -A "$1"
+  for aci in "$ROOT_DIR/result/"*.aci; do
```

Comment:
do we support multiple ACIs per result? otherwise we should assume we have only one and call `actool validate "$ROOT_DIR/result/"*.aci` directly